cumulus_etl/deid/ms-config.json

```diff
@@ -47,7 +47,6 @@
     {"path": "nodesByType('ContactDetail')", "method": "redact"},
     {"path": "nodesByType('ContactPoint')", "method": "redact"},
     {"path": "nodesByType('HumanName')", "method": "redact"},
-    {"path": "nodesByType('Identifier')", "method": "redact"},
```

Comment:
This was previously a blanket injunction against using `Identifier` elements, because I'd only seen them in `Resource.identifier` contexts, where the EHR stores PHI identifiers like MRNs or other such business-case oriented identifiers (I think in Patients, the SSN goes there? Or maybe that's an extension, I don't remember.)

Anyway. Device has a version.component field that for some reason uses the `Identifier` element, so I can't blanket-ban it now. But it is also never allow-listed anywhere else. This line was just a second line of defense against mistakes. But now that we have a real use case, we can drop this.